homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
@andrey-git didn't you say that you changed the frontend so that the brightness would show depending on supported features in state attributes? That attribute is always returned from state attributes, even when off.